manifests/init.pp

```diff
@@ -170,7 +170,11 @@
   validate_bool($experimental)
   validate_bool($localise_timestamp)
   validate_hash($extra_settings)
-  validate_bool($puppetdb_ssl_verify)
+  if $puppetdb_ssl_verify {
+    unless is_string($puppetdb_ssl_verify) {
+      validate_bool($puppetdb_ssl_verify)
```

Comment:
Bizarre. I suppose <code>is_string</code> is also validating the string in that case...